srcmedia/ts/components/ActiveFilters.ts

```diff
@@ -0,0 +1,271 @@
+import { Component } from '../lib/common'
+
+type Filter = {
+    type: 'checkbox' | 'range'
```

Comment:
Does boolean vs facet matter, or can you treat them both as checkboxes?

Reply:
in terms of interacting with them, both just checkboxes, yep. the only difference is in deciding what text goes in the button, as you noted below.